src/handlers/dns/DnsStreamHandler.h

```diff
@@ -376,6 +378,9 @@ class DnsStreamHandler final : public visor::StreamMetricsHandler<DnsMetricsMana
     mutable sigslot::signal<timespec> end_tstamp_signal;
     mutable sigslot::signal<const timespec> heartbeat_signal;
     mutable sigslot::signal<pcpp::Packet &, PacketDirection, pcpp::ProtocolType, uint32_t, timespec> udp_signal;
+
+    PcapInputEventProxy::UdpPredicate _udp_qname_predicate;
+    PcapInputEventProxy::UdpSignalCB _udp_qname_predicate_signal;
```

Comment:
this reference is not required anymore, right? we can only pass the lambda callback direct to the event proxy

Reply:
yes i think that's right